dashboards/traefik/traefik.json

```diff
@@ -222,7 +222,7 @@
             "nrqlQueries": [
               {
                 "accountId": 0,
-                "query": "FROM Metric SELECT\naverage(traefik_service_request_duration_seconds_sum) AS 'Average Request Duration (ms)'\nFACET instance, service\nLIMIT MAX\nWHERE instrumentation.name = 'remote-write'"
+                "query": "FROM Metric SELECT\naverage(traefik_service_request_duration_seconds_sum) AS 'Average Request Duration (ms)'\nFACET instance, service\nLIMIT MAX\nWHERE instrumentation.name = 'remote-write' AND prometheus_server IN ( {{prometheus_server}}) AND instance IN ( {{instance}} ) facet concat(prometheus_server,' - ',instance)"
```

Comment:
2 `FACET` conditions in this query; recommend consolidating into:

```
FACET concat(prometheus_server,' - ',instance) AS 'Instance', service
```

Reply:
Thanks, changes made.